Dockerfile

```diff
@@ -10,7 +10,13 @@ RUN mkdir /app && \
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
         apt-transport-https build-essential curl git libpq-dev python-dev \
-        default-libmysqlclient-dev gettext sqlite3 libffi-dev libsasl2-dev && \
+        default-libmysqlclient-dev gettext sqlite3 libffi-dev libsasl2-dev \
+        lsb-release curl gnupg && \
+    CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
+    echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
+    apt-get update -y && apt-get install google-cloud-sdk -y && \
+    apt-get remove -y curl lsb-release gnupg && \
```

Comment:
can we keep curl? and potentially install a text editor too for quick testing

Reply:
Ah yes I didn't see that `curl` was already being installed. I've also included `vim`.